### ISSUE TITLE
cluster.server is missing on macOS

### ISSUE THREAD

**Issue by carljmosca** [eclipse-che/che#14621]
Trying to start che on a Mac.

chectl server:start

Error: clusters[4].cluster.server is missing
    at /usr/local/lib/chectl/node_modules/@kubernetes/client-node/dist/config_types.js:19:19
    at Object._.map._.collect (/usr/local/lib/chectl/node_modules/underscore/underscore.js:205:24)
    at Object.newClusters (/usr/local/lib/chectl/node_modules/@kubernetes/client-node/dist/config_types.js:7:14)
    at KubeConfig.loadFromString (/usr/local/lib/chectl/node_modules/@kubernetes/client-node/dist/config.js:113:40)
    at KubeConfig.loadFromFile (/usr/local/lib/chectl/node_modules/@kubernetes/client-node/dist/config.js:85:14)
    at KubeConfig.loadFromDefault (/usr/local/lib/chectl/node_modules/@kubernetes/client-node/dist/config.js:178:22)
    at new KubeHelper (/usr/local/lib/chectl/lib/api/kube.js:24:21)
    at Start.<anonymous> (/usr/local/lib/chectl/lib/commands/server/start.js:86:20)
    at Generator.next (<anonymous>)
    at /usr/local/lib/chectl/node_modules/tslib/tslib.js:110:75

Mojave 10.14.6
Docker 2.1.3.0
Kubernetes 1.14.6
chectl/7.1.0 darwin-x64 node-v10.16.3

This is a computer on which I am running Docker/k8s and Helm without any other issues (of which I am aware).

**Comment by gorkem** [comment]
This seems to fail when it is trying to read the `kubeconfig`. Is your `kubeconfig` valid

**Comment by carljmosca** [comment]
I am sorry but I don't think I have enough experience with chectl to answer that question.

I know I can use kubernetes and helm for all other operations.  

The chectl directory in /usr/local/lib was newly created.

Without more context, I cannot answer.

Will remove chectl best I can and try again.

**Comment by carljmosca** [comment]
Uninstalled, reinstalled, same error.

Updated to "next", similar error, but referencing user home.

Error: clusters[4].cluster.server is missing
    at ~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/@kubernetes/client-node/dist/config_types.js:19:19
    at Object._.map._.collect (~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/underscore/underscore.js:205:24)
    at Object.newClusters (~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/@kubernetes/client-node/dist/config_types.js:7:14)
    at KubeConfig.loadFromString (~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/@kubernetes/client-node/dist/config.js:113:40)
    at KubeConfig.loadFromFile (~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/@kubernetes/client-node/dist/config.js:85:14)
    at KubeConfig.loadFromDefault (~/.local/share/chectl/client/0.0.20190920-next.4edc273/node_modules/@kubernetes/client-node/dist/config.js:178:22)
    at new KubeHelper (~/.local/share/chectl/client/0.0.20190920-next.4edc273/lib/api/kube.js:23:21)
    at new CheTasks (~/.local/share/chectl/client/0.0.20190920-next.4edc273/lib/tasks/che.js:24:21)
    at Start.<anonymous> (~/.local/share/chectl/client/0.0.20190920-next.4edc273/lib/commands/server/start.js:101:30)
    at Generator.next (<anonymous>)

**Comment by gorkem** [comment]
chectl requires `kubectl` or `oc` to be installed and authenticated to the cluster that you intend to work with che. It is how chectl gets the information for connecting kubernetes and run the setup.

More info on kubeconfig https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

**Comment by carljmosca** [comment]
I will look at this further but I don't know exactly what I am looking for quite yet.  Is it possible for you to point me to the step I have missed in the chectl docs?

We are currently using both kubectl and oc for our internally hosted OCP along with k8s running locally.  We're not currently seeing any problems switching as needed.

**Comment by carljmosca** [comment]
Same issue when I try after an oc login.

Went ahead and played around a bit with the online version.  Definitely better since last I tried but I will try again after things mature a bit more.

**Comment by gorkem** [comment]
Do you have a 'KUBECONFIG' environment variable set?

**Comment by carljmosca** [comment]
Have tried it w/ and w/o KUBECONFIG being set but given that mine is in the default location and I understand kubectl looks at the default location if not set I would not expect any different behavior.

**Comment by gorkem** [comment]
Do you really have a `cluster` definition without server on that file? The error message suggests that but it just does not make a lot of sense.

**Comment by carljmosca** [comment]
I have cleaned up the config file (there are multiple copies of multiple clusters which had not bother other apps we're running) and had to add a couple of parameters to the server:start command but I am not able to connect.  I don't know if the platform should be k8s or docker4mac given what I have read the latter is not yet supported.

As far as the domain parameter goes I have tried a combination of the local address with the nip.io and others.
